libraries/libfc/src/string.cpp

```diff
@@ -83,13 +85,42 @@ namespace fc  {
   std::string trim( const std::string& s )
   {
       return boost::algorithm::trim_copy(s);
-      /*
-      std::string cpy(s);
-      boost::algorithm::trim(cpy);
-      return cpy;
-      */
   }
 
+  std::pair<std::string&, bool> escape_str( std::string& str, escape_control_chars escape_ctrl,
+                                            std::size_t max_len, const char* add_truncate_str )
```

Comment:
I think the more modern version for the last parameter would be to be  be a `std::string_view` instead of a raw pointer.

Reply:
Done.